impl/src/bazel.rs

```diff
@@ -23,6 +23,35 @@ use crate::{
   util::RazeError,
 };
 
+use std::{
+  env,
+  path::{Path, PathBuf},
+};
+
+/** Returns a path to a Bazel workspace root if one could be found, otherwise None */
```

Comment:
You'd also want to document the behavior behind the assumption of the current_dir

Reply:
Updated.